.github/dependabot.yml

```diff
@@ -0,0 +1,60 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      # Standalone deps
+      - dependency-name: "mongodb"
+      - dependency-name: "bson"
+      # All the mongosh deps
+      - dependency-name: "@mongosh/node-runtime-worker-thread"
+      - dependency-name: "@mongosh/browser-repl"
+      - dependency-name: "@mongosh/logging"
+      - dependency-name: "@mongodb-js/oidc-plugin" # we bump mongosh when this updates
+      - dependency-name: "@mongodb-js/devtools-connect" # we bump mongosh when this updates
+      # All the electron and its related deps
+      - dependency-name: "electron"
+      - dependency-name: "@electron/remote"
+      - dependency-name: "@electron/rebuild"
+      - dependency-name: "browserslist"
+      - dependency-name: "node-abi"
```

Comment:
We have those as a separate script because there are some additional steps we need to take to make sure that this update doesn't mess up the monorepo, in particular electron dependency hoisting after version update. Just want to double-check: are we sure we can replace our update electron script with just dependabot?

Reply:
that's a good point that i missed. I don't think dependabot currently offers this. dropping this then makes sense. 

However what do you think about trying this out and see how it goes? (dependabot would eventually create a PR, which we can ignore, right?)

Reply:
Yeah, we can try that and see how it goes, no objections here!